src/users/views.py

```diff
@@ -181,6 +182,31 @@ def coc_agree(request):
         **reviews_state()._asdict(),
     })
 
+@login_required
+@require_GET
+def user_list(request):
```

Comment:
as our offline discussion, we should avoid exposing all the account sensitive data, and probably return 4XX for all the request without `role=reviewer`